lib/segment/src/index/sparse_index/sparse_vector_index.rs

```diff
@@ -26,28 +35,150 @@ impl<TInvertedIndex: InvertedIndex> SparseVectorIndex<TInvertedIndex> {
         id_tracker: Arc<AtomicRefCell<IdTrackerSS>>,
         vector_storage: Arc<AtomicRefCell<VectorStorageEnum>>,
         payload_index: Arc<AtomicRefCell<StructPayloadIndex>>,
-        path: PathBuf,
+        path: &Path, // TODO(sparse) use path to load/save index
```

Comment:
I propose to rebuild index while creation so that we have a persistency:
```
    pub fn open(
        id_tracker: Arc<AtomicRefCell<IdTrackerSS>>,
        vector_storage: Arc<AtomicRefCell<VectorStorageEnum>>,
        payload_index: Arc<AtomicRefCell<StructPayloadIndex>>,
        path: &Path, // TODO(sparse) use path to load/save index
    ) -> OperationResult<Self> {
        let searches_telemetry = SparseSearchesTelemetry::new();
        let path = path.to_path_buf();
        let s = Self {
            id_tracker,
            vector_storage,
            payload_index,
            path,
            inverted_index: Default::default(),
            searches_telemetry,
            indexed_count: 0,
        };
        s.build_index(false)?; // TODO(sparse) load from disk
        Ok(s)
    }
```

Reply:
I went to extra mile and added open to the inverted index trait to load properly mmaped index
https://github.com/qdrant/qdrant/pull/2900/commits/ff69e3ad9969cb03defefc987ae71f9bcb9617aa

I will add in the future unit test for using the `SparseVectorIndex` is a mmaped inverted index.